affiliated/index.html

```diff
@@ -218,7 +218,10 @@ <h1 id="becoming-an-affiliated-package">Becoming an Affiliated Package<a class="
 		<li>Be potentially useful to astronomers.  This can mean useful to a
 		specific sub-domain of astronomy, or more broadly useful to a large
 		fraction of astronomy (or beyond, as long as it is also useful for
-		astronomy).
+		astronomy).</li>
+
+		<li>Specifically use, interface with, or provide complimentary capabilities
```

Comment:
compliment or complement? 🤔 
```suggestion
		<li>Specifically use, interface with, or provide complementary capabilities
```

Reply:
either.